docs/reference/announcements-release-notes/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+id: overview
+title: "Announcements & release notes"
+description: "Important announcements and release notes for upcoming and past Camunda 8 releases that customers should be aware of."
+---
+
+In this section you will find announcements and release notes for each supported version of the Camunda documentation.
+
+**Announcements** include:
+
+- Deprecations and breaking changes
+- Supported environments (updates regarding OpenJDK, Spring Boot, Elasticsearch, Keycloak, Amazon OpenSearch, Docker tags, Helm chart versions, etc.)
```

Comment:
Can we please be a bit more generic about the "changes". Reason behind: we are having many breaking changes with 8.8, but this should/must go down afterwards. So I want us to have more flexibility with the wording.
E.g.,:

```suggestion
- Key changes (e.g., deprecations)
- Changes in supported environments (updates regarding OpenJDK, Spring Boot, Elasticsearch, Keycloak, Amazon OpenSearch, Docker tags, Helm chart versions, etc.)
```

Reply:
Agreed -- adjusting in incoming commit.